### PR TITLE
update CDAP label and description

### PR DESCRIPTION
**Pull request by dereklwood**
update CDAP label and description per marketing.  These show up in the "Add Service" wizard.

### CODE REVIEW

File: src/descriptor/service.sdl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "CDAP",
-  "label": "Cask DAP",
-  "description": "Cask Data Application Platform",
+  "label": "CDAP",
+  "description": "The Cask Data Application Platform (CDAP) is an easy-to-use, open source and enterprise-ready integrated platform for organizations to build, deploy and operate data-driven applications.",
```
Comment (awholegunch) [inline src/descriptor/service.sdl]: Add an "Oxford comma": 
```
to build, deploy, and operate data-driven applications.

**Comment by awholegunch** [comment]
This will require re-doing all the screen shots in the documentation for the installation, as they show the text "Cask DAP".

**Comment by wolf31o2** [comment]
Also, we'll need to do the same thing for the Ambari service.

**Comment by sreevatsanraman** [comment]
👍

**Comment by dereklwood** [comment]
opened https://issues.cask.co/browse/CDAP-6190 for updating the screenshots

**Pull request merged**